firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -21,37 +21,38 @@
 import com.google.firebase.FirebaseApp;
 
 /**
- * A layer that locally caches a few Firebase Installation attributes on top the Firebase
- * Installation backend API.
+ * A layer that locally persists a few Firebase Installation attributes on top the Firebase
+ * Installation API.
  */
-public class FiidCache {
-  // Status of each cache entry
-  // NOTE: never change the ordinal of the enum values because the enum values are stored in cache
+public class PersistedFid {
+  // Status of each presisted fid entry
+  // NOTE: never change the ordinal of the enum values because the enum values are stored in shared
+  // prefs
   // as their ordinal numbers.
-  public enum CacheStatus {
-    // Cache entry is synced to Firebase backend
+  public enum PersistedStatus {
```

Comment:
seems like this enum should be called "RegistrationStatus"

Reply:
+1